### PR TITLE
[LowestPriceBeforeTheDiscount][UI][Admin] Displaying information

### PR DESCRIPTION
**Pull request by Rafikooo**
I'm aware that these do not look perfect, we can discuss it

image
image

### CODE REVIEW

File: tests/Behat/Element/Admin/Product/ShowPage/VariantsElement.php [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+        );
+
+        if (
+            $variant->find('css', '.content .variant-name')->getText() === $name &&
+            $variantContent->find('css', sprintf('tr.pricing:contains("%s") td:nth-child(4)', $channel))->getText() === $lowestPriceBeforeDiscount
+        ) {
+            return true;
+        }
+
+        return false;
```
Comment (GSadee) [inline tests/Behat/Element/Admin/Product/ShowPage/VariantsElement.php:67]:
```suggestion
        return
            $variant->find('css', '.content .variant-name')->getText() === $name &&
            $variantContent->find('css', sprintf('tr.pricing:contains("%s") td:nth-child(4)', $channel))->getText() === $lowestPriceBeforeDiscount
        ;
```

File: tests/Behat/Element/Admin/Product/ShowPage/VariantsElementInterface.php [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+declare(strict_types=1);
+
+namespace Tests\Sylius\PriceHistoryPlugin\Behat\Element\Admin\Product\ShowPage;
+
+interface VariantsElementInterface
+{
+    public function hasProductVariantWithLowestPriceBeforeDiscountInChannel(
+        string $productVariantName,
+        string $lowestPriceBeforeDiscount,
+        string $channel,
```
Comment (GSadee) [inline tests/Behat/Element/Admin/Product/ShowPage/VariantsElementInterface.php:21]:
```suggestion
        string $channelName,
```

File: tests/Behat/Resources/suites/ui/product/viewing_products.yaml [hunk 3]
```diff
@@ -22,7 +25,9 @@ default:
                 - Tests\Sylius\PriceHistoryPlugin\Behat\Context\Setup\ChannelContext
                 - Tests\Sylius\PriceHistoryPlugin\Behat\Context\Setup\ProductContext
 
+                - sylius.behat.context.ui.admin.product_showpage
```
Comment (GSadee) [inline tests/Behat/Resources/suites/ui/product/viewing_products.yaml:28]: IMO these new scenarios should be in a different suite, as this one is intended for shop

Comment (Rafikooo) [reply tests/Behat/Resources/suites/ui/product/viewing_products.yaml:28]: We decided not to touch it as it is mixed in Open Source

**Review by GSadee — APPROVED** [review]

**Comment by GSadee** [comment]
Thanks, Rafał! :tada:

**Pull request merged**